### PR TITLE
Add documents

### LINKED ISSUE #10: Document and tutorial
- README.md
- tutorial document

### PR DESCRIPTION
**Pull request by nukosuke**
resolve #10 

This is orphan branch of docs [Jekyll](https://jekyllrb.com/) project. Jekyll is static site generator written in Ruby.
Don't mind CI fail because it is result for trying to build based on config.yml for golang project (master branch).

After merge this PR, the docs can be seen at https://xflagstudio.github.io/zenform/.

### CODE REVIEW

**Comment by nukosuke** [comment]
# top page
Screenshot from Gyazo

# docs/installation
[Screenshot from Gyazo](https://gyazo.com/a99275c87c27d4f0946520ad286b954d)

# docs/credentials
[Screenshot from Gyazo](https://gyazo.com/d0e657ca5462a63889236bdee1705820)

# docs/ticket-field
[Screenshot from Gyazo](https://gyazo.com/0a0917f747a1f258585411b9c3d11009)

File: _docs/21_ticket_field.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+|:------------------------------|:--------------------------------------|:----------------------------------------------------------|
+| `slug`                        | string                                | Unique identifier of the ticket field                     |
+| `title`                       | string                                | Title of the ticket field                                 |
+| `visible_in_portal`           | boolean<br/>( `"TRUE"` \| `"FALSE"` ) | Whether the ticket field is visible on portal site        |
+| `editable_in_portal`          | boolean<br/>( `"TRUE"` \| `"FALSE"` ) | Whether the ticket field is editable on portal site       |
+| `required_in_portal`          | boolean<br/>( `"TRUE"` \| `"FALSE"` ) | Whether the ticket field is mandatory item on portal site |
+| `description`                 | string                                | Description of the ticket field                           |
+| `custom_field_options.names`  | []string                              | option name to custom                                     |
```
Comment (upscent) [inline _docs/21_ticket_field.md]: > option name to custom

In capitals?

Comment (nukosuke) [reply _docs/21_ticket_field.md]: Fixed in a872a4b

**Comment by nukosuke** [comment]
Thanks guys!

**Pull request merged**